plugins/eclipse/ui/pom.xml

```diff
@@ -26,19 +22,19 @@
       <dependency>
         <groupId>com.mobidevelop.robovm</groupId>
         <artifactId>robovm-dist-compiler</artifactId>
-        <version>${robovm.version}</version>
```

Comment:
This causes the sed failure. It expects the old `robovm.version` https://github.com/MobiVM/robovm/blob/master/release.sh#L61